packages/@react-aria/textfield/src/useTextField.ts

```diff
@@ -110,23 +113,30 @@ export function useTextField<T extends TextFieldIntrinsicElements = DefaultEleme
     isDisabled = false,
     isRequired = false,
     isReadOnly = false,
-    validationState,
-    isInvalid = false,
     type = 'text',
-    value,
-    defaultValue,
-    onChange = () => {}
+    validationBehavior = 'aria'
   }: AriaTextFieldOptions<TextFieldIntrinsicElements> = props;
+  let [value, setValue] = useControlledState<string>(props.value, props.defaultValue || '', props.onChange);
```

Comment:
this is a bit odd, i haven't seen useControlledState in an aria package before. I assume this is because textfield doesn't have a corresponding stately hook? should it? I guess that would be breaking...